addon/mixins/sortable-item.js

```diff
@@ -628,6 +628,10 @@ export default Mixin.create({
    * @private
    */
   _drag(dimension) {
+    if(Ember.get(this, 'isDestroyed') || Ember.get(this, 'isDestroying')){
```

Comment:
Sorry - I had typed some other comments that didn't get submitted correctly. Can we use `this.get` instead of `Ember.get` to maintain code consistency?